docs/modules/casper.rst

```diff
@@ -386,6 +386,26 @@ Default wait timeout, for ``wait*`` family functions.
 ``Casper`` prototype
 ++++++++++++++++++++
 
+``addPseudoClass()``
+-------------------------------------------------------------------------------
+
+**Signature:** ``addPseudoClass(String pseudoClass, String selector)``
+
+.. versionadded:: 1.1
+
+Adds a speudo class to simulate css hover style and other speudo class::
+
```

Comment:
Adds a ~~speudo~~ *pseudo* class to simulate css hover style and other *pseudo* class *es*.

Needs a global ```s/speudo/pseudo/```